_posts/2024-07-19-jberet-3_0_0_beta1-release.md

```diff
@@ -0,0 +1,38 @@
+---
+layout:     post
+title:      "The Release Note Of JBeret 3.0.0.Beta1"
+subtitle:   ""
+date:       2024-07-19
+author:     Wei Nan Li
+---
+
+Recently JBeret has a major version release::
+
+- [Release v3.0.0.Beta1 · jberet/jsr352](https://github.com/jberet/jsr352/releases/tag/3.0.0.Beta1)
+
+In this release it include these major changes:
+
+- [JBERET-445: JUnit5 Migration #467](https://github.com/jberet/jsr352/pull/467)
+- [JBERET-472: Extract database props from env-vars #473](https://github.com/jberet/jsr352/pull/473)
+
+And it added the tests against main branch of WildFly server, both preview and non-preview ones:
+
+- [JBERET-599: Setup the CI testings against WildFly main branch on #490](https://github.com/jberet/jsr352/pull/490)
+- [Add WildFly Preview CI Testing. #531](https://github.com/jberet/jsr352/pull/531)
+
+In addition, it removes some unnecessary API dependencies:
+
+- [Remove the dependency of jakarta.persistence-api #536](https://github.com/jberet/jsr352/pull/536)
+- [Remove the dependency of the validation-api #540](https://github.com/jberet/jsr352/pull/540)
+
+And it contains these Jakarta API related components upgrades::
```

Comment:
There is a double "::" here. Should you also mention something about the upgrades that they were done in preparation for Jakarta EE 11?

Reply:
okay!